### PR TITLE
Version 1.1.9

### PR DESCRIPTION
**Pull request by SC426**
sphinxcontrib-serializinghtml 1.1.9

**Destination channel:** defaults

### Links

- [PKG-3747](https://anaconda.atlassian.net/browse/PKG-3747) 
- [Upstream repository](https://github.com/sphinx-doc/sphinxcontrib-serializinghtml)
- [Upstream changelog/diff](https://github.com/sphinx-doc/sphinxcontrib-serializinghtml/blob/master/CHANGES)
- Relevant dependency PRs: [PKG-3133](https://anaconda.atlassian.net/browse/PKG-3133)

### Explanation of changes:

- bump version to 1.1.9
- remove noarch build
- re-add pip check 
- cleanup and linter fixes

[PKG-3747]: https://anaconda.atlassian.net/browse/PKG-3747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PKG-3133]: https://anaconda.atlassian.net/browse/PKG-3133?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools
+    - wheel
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
```
Flit replaces setuptools here. So setuptools is not needed. See https://inspector.pypi.io/project/sphinxcontrib-serializinghtml/1.1.9/packages/5c/41/df4cd017e8234ded544228f60f74fac1fe1c75bdb1e87b33a83c91a10530/sphinxcontrib_serializinghtml-1.1.9.tar.gz/sphinxcontrib_serializinghtml-1.1.9/pyproject.toml#line.3.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
-  commands:
-    # pip check incorrectly shows dependencies for sphinx 4.0.1
-    #- pip check
   imports:
     - sphinxcontrib
     - sphinxcontrib.serializinghtml
+  requires:
+    - pip
+  commands:
+    - pip check
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:32]: We could easily run the tests: `pytest --pyargs sphinxcontrib.serializinghtml -v`

File: recipe/meta.yaml (hunk 1 above)
Comment (SC426) [reply recipe/meta.yaml]: This change has been made

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml:32]: This change has been made

Comment (SC426) [reply recipe/meta.yaml:32]: Reverted this since pytest fails

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by cbouss — APPROVED** [review]

**Pull request closed without merging**